packages/backend/server/src/core/config/config.ts

```diff
@@ -69,4 +70,9 @@ defineModuleConfig('flags', {
     desc: 'Only allow users with early access features to access the app',
     default: false,
   },
+  allowDemoWorkspace: {
+    desc: 'Allow guests to use the demo workspace',
+    default: true,
+    env: ['AFFINE_ALLOW_DEMO_WORKSPACE', 'boolean'],
```

Comment:
only essential parameters such as database connections should be stored in environment variables
other variables should be configured only through config.json